includes/models/model.llms.course.php

```diff
@@ -16,9 +16,10 @@
  * @since 1.0.0
  * @since 3.30.3 Explicitly define class properties.
  * @since 4.0.0 Remove previously deprecated class methods.
- * @since [version] Check for an empty sales page URL or ID.
+ * @since [version] Add `__construct()`.
+ *              Move audio and video embed properties and methods to `LLMS_Trait_Audio_Video_Embed`.
+ *              Move sales page properties and methods to `LLMS_Trait_Sales_Page`.
```

Comment:
New method addition is unnecessary in the class docs (the addition of the method can be seen on the method's changelog inherently)

```suggestion
 * @since [version] Move audio and video embed properties and methods to `LLMS_Trait_Audio_Video_Embed`.
 *              Move sales page properties and methods to `LLMS_Trait_Sales_Page`.
```